pkg/asset/tls/helper.go

```diff
@@ -15,6 +16,7 @@ const (
 )
 
 func assetFilePath(filename string) string {
+	logrus.Debugf("assetFilePath: %s", filepath.Join(tlsDir, filename))
```

Comment:
Reminder: to be removed in the final version

Reply:
Done